jython/jmri_bindings.py3

```diff
@@ -0,0 +1,46 @@
+import jmri as jmri
```

Comment:
When/how is this file used? I'm asking to learn how things work.

Reply:
See below for a description of the problem that this is working around.